mrtt-ui/src/library/auth/interceptor.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios'
+
```

Comment:
There might be a better less implied, more explicit way to do this, but we can add that later as the app grows. (im thinking using acios.create to create an instance that has the tokens set that is named something like axiosWithAuthentication or something. )

For now, can rename this function to initializeAxiosAuthenticationInterceptor?

Reply:
Addressed in https://github.com/globalmangrovewatch/gmw-users/pull/157/commits/5beb14d022c0ab58e6ba97aa47136142d8bce995